.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -61,6 +61,9 @@ Execute `sudo apt-get update && sudo apt-get upgrade` on
 - [ ] Check that account linking via e-mail verification works correctly
 - [ ] Check that account linking via username & password verification works correctly
 
+## Check e-mails sent by backend
+- [ ] Send an invitation request from one of the dev servers and check if the e-mail response contains the right attachments and is displayed correctly.
```

Comment:
Wenn ich in Firefox das drop down der user settings öffne ist der erste Eintrag weiß und wird erst nach einem Klick orange, zudem funktioniert die Seite scheinbar im Edge gar nicht (security issues)

Reply:
In Chrome sieht das dropdown menu wie erwartet aus. In Edge ist es so wie Siegfried für Firefox beschrieben hat. 
Das Problem mit den Zertifikaten, wenn man die Seite über localhost aufruft konnte ich bei mir nicht reproduzieren.
An der Styleanpassung für die Requests Seite arbeite ich gerade.

Reply:
Das ist das letzte offene Ticket. Das ist beim Merge mit main passiert.

Reply:
Komischerweise ist das Menü bei mir in Chrome jetzt auch nicht mehr wie erwartet

Reply:
Die Sachen sind gefixed.

Reply:
Ja passt jetzt